post-install.xql

```diff
@@ -71,13 +73,25 @@ declare function local:create-user() as empty-sequence() {
             else util:log-system-out(concat('unable to read from file "', normalize-space(environment-variable($opt)), '"'))
         else if($opt = 'MERMEID_mermeid_password') then 
             string(environment-variable($opt))
-            else ()
+        else "mermeid"
 
     return if ($password) then 
         sm:create-account('mermeid', $password, 'mermeid', ('mermedit'))
     else ()
 };
 
+
```

Comment:
is this function really necessary? I thought the variable "MERMEID_app_url" would be handled by the generic function `local:set-options()` (in that same file) and thus updating the properties file?

Reply:
oh right, you can just use MERMEID_exist_endpoint, so essentially this is more of a documentation issue.